docs/content/doc/usage/email-setup.en-us.md

```diff
@@ -76,12 +76,15 @@ The following configuration should work with GMail's SMTP server:
 ```ini
 [mailer]
 ENABLED        = true
+HOST           = smtp.gmail.com:465
```

Comment:
IIRC, this option was deprecated by  #18982 , do not use it anymore.

Reply:
Thanks for the context! Let me test this on my instance and I'll apply the change.

Reply:
Just checked and it works perfectly without the `HOST` setting.